cf-ec2-instance-template.yml

```diff
@@ -0,0 +1,64 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description:  This template creates EC2 Instance inside a public A subnet of Bison Network.
+Parameters:
+    InstanceType:
+        Type: String
+        Description: EC2 instance type
+        Default: t2.micro
+        AllowedValues:
+            - t2.micro
+    AMI:
+        Type: String
```

Comment:
It's better to use AWS specific type here: AWS::EC2::Image::Id

Reply:
Fixed. Thanks.